### ISSUE TITLE
Training file error when using initModel

### ISSUE THREAD

**Issue by Tiriar** [facebookresearch/StarSpace#173]
Hello, I am trying to train a StarSpace classifier while initializing the algorithm with a fastText model (which I converted to a tsv format). When I try to run StarSpace on the training file without the initModel option, everything runs ok:
```
./starspace train -trainFile training.txt -model wiki
Arguments: 
lr: 0.01
dim: 100
epoch: 5
maxTrainTime: 8640000
saveEveryEpoch: 0
loss: hinge
margin: 0.05
similarity: cosine
maxNegSamples: 10
negSearchLimit: 50
thread: 10
minCount: 1
minCountLabel: 1
label: __label__
ngrams: 1
bucket: 2000000
adagrad: 1
trainMode: 0
fileFormat: fastText
normalizeText: 0
dropoutLHS: 0
dropoutRHS: 0
Start to initialize starspace model.
Build dict from input file : training.txt
Read 0M words
Number of words in dictionary:  395
Number of labels in dictionary: 105
Loading data from file : training.txt
Total number of examples loaded : 608
Training epoch 0: 0.01 0.002
Epoch: 98.5%  lr: 0.010000  loss: 0.025505  eta: <1min   tot: 0h0m0s  (19.7%)
 ---+++                Epoch    0 Train error : 0.01774764 +++--- ☃
Training epoch 1: 0.008 0.002
Epoch: 98.5%  lr: 0.008000  loss: 0.000451  eta: <1min   tot: 0h0m0s  (39.7%)
 ---+++                Epoch    1 Train error : 0.00089402 +++--- ☃
Training epoch 2: 0.006 0.002
Epoch: 98.5%  lr: 0.006000  loss: 0.000014  eta: <1min   tot: 0h0m0s  (59.7%)
 ---+++                Epoch    2 Train error : 0.00013078 +++--- ☃
Training epoch 3: 0.004 0.002
Epoch: 98.5%  lr: 0.004000  loss: 0.000107  eta: <1min   tot: 0h0m0s  (79.7%)
 ---+++                Epoch    3 Train error : 0.00006586 +++--- ☃
Training epoch 4: 0.002 0.002
Epoch: 98.5%  lr: 0.002000  loss: 0.000005  eta: <1min   tot: 0h0m0s  (99.7%)
 ---+++                Epoch    4 Train error : 0.00002807 +++--- ☃
Saving model to file : wiki
Saving model in tsv format : wiki.tsv
```
However, when I use the same with the initModel, it looks like the model is loaded fine, but for some reason the algorithm throws and exception for the training file:
```
./starspace train -trainFile training.txt -model wiki -initModel wiki_50k.tsv -dim 300
Arguments: 
lr: 0.01
dim: 300
epoch: 5
maxTrainTime: 8640000
saveEveryEpoch: 0
loss: hinge
margin: 0.05
similarity: cosine
maxNegSamples: 10
negSearchLimit: 50
thread: 10
minCount: 1
minCountLabel: 1
label: __label__
ngrams: 1
bucket: 2000000
adagrad: 1
trainMode: 0
fileFormat: fastText
normalizeText: 0
dropoutLHS: 0
dropoutRHS: 0
Start to load a trained embedding model in tsv format.
Loading dict from model file : wiki_50k.tsv
Number of words in dictionary:  50016
Number of labels in dictionary: 0
Loading model from file wiki_50k.tsv
Model loaded.
Loading data from file : training.txt
Total number of examples loaded : 0
ERROR: File 'training.txt' does not contain any valid example.
Please check: is the file empty? Do the examples contain proper feature and label according to the trainMode? If your examples are unlabeled, try to set trainMode=5.
```
Any idea why this happens?

**Comment by ledw** [comment]
@Tiriar Hi, thanks for reporting. Yes, StarSpace does not add new vector once the initModel is set, so that's why it gives error since there's no label presented in initModel. I'd suggest you to add some dummy / random vectors as your label vector in addition to what you have from fastText. That way StarSpace can carry on training from there.

**Comment by Tiriar** [comment]
@ledw Thank you, adding the label vectors to the pretrained model works :+1: . Closing the issue.

**Comment by luthfianto** [comment]
@Tiriar @ledw how to add the training label vectors?

**Comment by ellabee7** [comment]
Sorry @rilut , this was over 2 years back and I do not remember how exactly I did it. But as long as you have the file with pre-trained vectors in txt/tsv format, adding new vectors should be as simple as adding new rows to the file - just generate a random vector for each label and add them as new rows in the same format as the other word vectors that are already present.

**Comment by luthfianto** [comment]
@Tiriar no worries, thank you
